webpack/webpack.common.js

```diff
@@ -0,0 +1,49 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+module.exports = {
+    // entry: {
+    //     main: "./src/index.js",
+    //     app: "./src/components/App.js",
+    //     LogIn: "./src/components/AuthPage/LogIn.jsx",
+    //     SignUp: "./src/components/AuthPage/SignUp.jsx",
+    //     MainPage: "./src/components/MainPage/MainPage.jsx",
+    //     Profile: "./src/components/ProfilePage/Profile.jsx",
+    //     LinkDetails: "./src/components/ProfilePage/LinkDetails.jsx",
+    //     ShortenerPage: "./src/components/ShortenerPage/ShortenerPage.jsx",
+    //     Search: "./src/components/SearchLinksPage/SearchLinksPage.jsx",
+    //     SearchedLinkDetails: "./src/components/SearchLinksPage/SearchedLinkDetails.jsx",
+    // },
+    entry: './src/index.js',
+    output: {
+        path: path.join(__dirname, "..", "/dist"),
+        filename: "[name].bundle.js",
```

Comment:
name of chunks should be different for dev and prod build.
Prod build shouldn't contain real names of chunks.

Also, please read about webpack substitution like '[hash], [contenthash], etc'